active/fundraising.md

```diff
@@ -2,31 +2,60 @@
 
 ## Scope of responsibilities
 
-Coordinates fundraising efforts, particularly around corporate and major donations.
+This working group coordinates fundraising efforts, particularly around corporate and major donations.
+
+The goals of the group are:
+
+- Acquire funding to secure the financial future of Django and support future growth of Django
+- Build and strengthen relationships between Django and donor sources
+- Raise awareness of the Django Web Framework and its community with the broader world
+
+Delegated responsibilities:
+
+- Contacting, developing and implementing financial sponsorships with donor sources
+- Clarifying and elaborating on the various creative strategies that donor sources of different sizes and types can use to support Django
 
-Delegated responsibilities TBD.
 
 ## Initial membership
 
 - Chair: TBD
 - Co-Chair: TBD
 - Board Liaison (must be an active Board member; may be the same as Chair/Co-Chair): TBD
-- Other members: TBD
+- Other members:
+  - Jacob Kaplan-Moss
+  - Priya Pahwa
+  - Thibaud Colas
+  - Tom Carrick
+  - Sarah Abderemane
+  - Stephanie Goulet
+
 
 ## Future membership
 
 Direct membership: new members may self-nominate; the WG will vote (50%+1) to approve/deny new members; the WG will directly vote on new Chair/Co-Chairs.
 
-This committee is currently forming and seeking volinteers, including chair and co-chair. 
+This committee is currently forming and seeking volunteers, including chair and co-chair. 
+
+### Navigating Conflicts of Interest
+Each new member must disclose any potential conflicts of interest. These disclosures will be made transparent by means of ____. 
+
+Any person with a conflict of interest will need to recuse themselves from any votes or other decision-making related to their conflict.
+
+No more than X% of the Fundraising Working Group may contain individuals associated with the same organization
```

Comment:
I think this decision needs to be made by a group of people, not just me giving a number.

Reply:
I don’t understand why this would be enough of an issue to call out here?

Reply:
I think it's fair to be forward thinking about what this may turn into. Some of the larger employers are going to be ideal funding sources. It seems weird to permit the WG the possibility to have a voting bloc for that company.

I think it's wiser to take a more conservative stance from the start. It can be scaled back if it becomes a problem in the future. However, the reverse situation seems messier to untangle.

Reply:
I can see the problem but am not convinced it’s worth having to define this in that clear-cut of a way. Would rather replace this with a line in "future membership" about having individuals from a good diversity of different organizations perhaps. 

The reverse situation – if it comes to that, the board can always decide to shut down a working group (requires a board vote).

Reply:
Could you elaborate on what problems including this would invite?

Reply:
Well first we have to agree on a X% figure, which to me feels like a distraction compared to other aspects of getting this group up and running. Then I see it being a distraction in the long term too when selecting new members.

For example, right now 50% of the group’s draft membership is associated with Djangonaut Space. If we’re concerned about voting bloc then that’s a problem at least the way that rule is written right now? It seems entirely plausible the group’s membership could have 50% DEFNA members, or Django Girls, or Wagtail; and consider adding one more. All of which wouldn’t be ideal but if it’s the right person at the right time, I’d rather not have to think about a hard and fast "X%".

Reply:
Thank you Thibaud, that helps me understand your perspective. You're right that the way it's written now would be problematic with the proposed members. That needs a change. Though since I'm not a proposed member and one of your concerns is that this is distracting, I'll leave it to the remainder members to voice their opinions.